rules-tests/Transform/Rector/Attribute/AttributeKeyToClassConstFetchRector/Fixture/not_replacing_existing_constant.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\Transform\Rector\Attribute\AttributeKeyToClassConstFetchRector\Fixture;
+
+use Doctrine\DBAL\Types\Types;
```

Comment:
I would like to use existing constants so the test actually tests something.
Can you point me in the right direction where to find a class with real constants, that is safe to use?